superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -164,10 +166,14 @@ export function useListViewResource<D extends object = any>(
     hasPerm,
     fetchData,
     toggleBulkSelect,
-    refreshData: () => {
+    refreshData: (mineConfig: FetchDataConfig | null = null) => {
```

Comment:
nits: 
- can we name this something more generic, `mineConfig` confused me when I read it before seeing the lines below, how about `providedConfig` ?
- you can make the arg optional and you won't need the ` | null = null` portion

Reply:
Yes!